### ISSUE TITLE
bug: incorrect parsing of json tags in Go structs if omitempty is used

### ISSUE THREAD

**Issue by TomCranitch** [windmill-labs/windmill#1427]
### Describe the bug

 If you use the [omitempty tag](https://pkg.go.dev/encoding/json#Marshal) (e.g. ``Field int `json:"myName,omitempty"` ``) then the field name is parsed to be `omitempty`. 

I think the cause is the use of `.last()` on this line: https://github.com/windmill-labs/windmill/blob/b8b4308aef3d0b3446a1a5c7982bbc0665e401ae/backend/parsers/windmill-parser-go/src/lib.rs#L99

### To reproduce

1. Write a go script with the following code:
```
func main(s struct {
  NameOne string `json:"nameOne,omitempty"`
  NameTwo string `json:"nameTwo,omitempty"`
}) {
  return s, nil
}
```
 2. The parsed arguments on the right will have a struct with a single element called `omitempty`

### Expected behavior

The output is a struct with two element `nameOne`, `nameTwo`

### Application version

Windmill v1.87.0-35-gb75b28c3

**Comment by rubenfiszel** [comment]
Thanks for the report, I will attempt a fix tonight

**Comment by rubenfiszel** [comment]
Fixed in: [95e52b2](https://github.com/windmill-labs/windmill/commit/95e52b2d74f8f752ea83b89e547f636b569f8771)
